packages/http-client/src/ceramic-http-client.ts

```diff
@@ -72,17 +78,18 @@ export class CeramicClient implements CeramicApi {
   public readonly pin: PinApi
   public readonly admin: AdminApi
   public readonly index: IndexApi
-  public readonly context: Context
+  private _signer: CeramicSigner
+  private _did?: DID
```

Comment:
same as for `Ceramic`.  Can we remove the need to store both the `_signer` AND the `_did`?  If not, let's at least add comments here too as to why we have both and when developers should use which one.